libbeat/docs/breaking.asciidoc

```diff
@@ -9,9 +9,152 @@ changes, but there are breaking changes between major versions (e.g. 5.x to
 
 See the following topics for a description of breaking changes:
 
+* <<breaking-changes-6.3>>
 * <<breaking-changes-6.0>>
 * {auditbeat}/auditbeat-breaking-changes.html[Breaking changes in Auditbeat 6.2]
 
+[[breaking-changes-6.3]]
+=== Breaking changes in 6.3
+
+This section discusses the main changes that you should be aware of if you
+upgrade the Beats to version 6.3. Please also review the relevant
+Breaking Changes sections of the <<release-notes,release notes>>.
+
+[[breaking-changes-mapping-conflict]]
+==== New `host` namespace may cause mapping conflicts for Logstash
+
+// REVIEWERS: I describe these as mapping conflicts throughout, but the acutal
+// error message says mapper parsing exception. I'm wondering if I should use
+// that instead?
+
+This breaking change applies only to users who send Beats events to Logstash.
```

Comment:
And users using their own template.

Reply:
@ph You're saying that this issue applies to Beats users who use a custom template and aren't using Logstash, correct? If they have `host` field defined in their custom template, they'll run into mapping errors, no?

Reply:
@dedemorton 
It applies to the following use cases:

- FB -> LS,  using the default logstash-output-elasticsearch template.
- FB -> LS,  using their own template templates with the host defined.

Reply:
@dedemorton Yes this is true, so let's keep it more specific we can adjust if people are posting on discuss.